presto-product-tests/src/main/java/io/prestosql/tests/hive/TestExternalHiveTable.java

```diff
@@ -127,6 +136,21 @@ public void testDeleteFromExternalPartitionedTableTable()
         assertThat(onPresto().executeQuery("SELECT * FROM " + EXTERNAL_TABLE_NAME)).hasRowsCount(0);
     }
 
+    @Test(groups = {HIVE_WITH_EXTERNAL_WRITES, PROFILE_SPECIFIC_TESTS})
```

Comment:
Nit: I think adding the group name to TestGroups can also be done to be future proof and not break the structure of having every TestGroup listed in the TestGroups class.

Reply:
I am not sure i understand. please explan

Reply:
Oh, nevermind. I see that the change was kept. I was talking about using `TestGroups.HIVE_WITH_EXTERNAL_WRITES`. But it's already there https://github.com/prestosql/presto/pull/4532/files#diff-18c1150a83e0fa5ed4a00019bfa438a2R32.

:stuck_out_tongue: